.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
 .vscode/
-node_modules/
```

Comment:
Why has node_modules been removed? I'm assuming its just because we have no package.json?

Reply:
Yeah, it's simply unnecessary to have it, since this package will probably never have any Node packages.

Reply:
Seems sensible.